code-challenges/chaining/challenges-10.test.js

```diff
@@ -0,0 +1,194 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named count that, given an integer and an array of arrays, uses either filter, map, or reduce to count the amount of times the integer is present in the array of arrays.
+
+Note: You might need to use the same method more than once.
+
+For example, count(5, [[1, 3, 5, 7, 9], [5, 5, 5], [1, 2, 3]]) returns 4.
+------------------------------------------------------------------------------------------------ */
+
+const count = (target, input) => {
+  // Solution code here...
+  let count = 0;
+  input.forEach(arr => {
+    arr.map(element => element === target ? count++ : element);
```

Comment:
You're using map which return an array, but not returning an array, another forEach might have been more appropriate